src/components/navigation/Logo.tsx

```diff
@@ -0,0 +1,21 @@
+import { styled } from '@atom-learning/components'
+import * as React from 'react'
+
+type LogoProps = React.ComponentProps<typeof StyledSVG>
+
+const StyledSVG = styled('svg', {})
+
+export const Logo: React.FC<LogoProps> = React.forwardRef((props, ref) => (
```

Comment:
Wouldn't be nicer to use the new logo, instead of the current one? it is just so much nicer :)

Reply:
Yeah it was initially before I took it back to the current one, for a few reasons:
- I didn't want to use it without pairing it with the updated brand
- I also didn't want the DS to be the first and only public outing of the new logo

Reply:
fair enough, it will be a simple switch anyway.